types/webxr/index.d.ts

```diff
@@ -1030,22 +1030,6 @@ declare enum XOVR_multiview2 {
     FRAMEBUFFER_INCOMPLETE_VIEW_TARGETS_OVR = 0x9633,
 }
 
-interface OVR_multiview2 {
-    readonly FRAMEBUFFER_ATTACHMENT_TEXTURE_NUM_VIEWS_OVR: 38448;
-    readonly FRAMEBUFFER_ATTACHMENT_TEXTURE_BASE_VIEW_INDEX_OVR: 38450;
-    readonly MAX_VIEWS_OVR: 38449;
-    readonly FRAMEBUFFER_INCOMPLETE_VIEW_TARGETS_OVR: 38451;
-
-    framebufferTextureMultiviewOVR(
-        target: GLenum,
-        attachment: GLenum,
-        texture: WebGLTexture,
-        level: number,
-        baseViewIndex: number,
-        numViews: number,
-    ): WebGLRenderbuffer;
-}
-
 declare abstract class OVR_multiview2 implements OVR_multiview2 {}
```

Comment:
Might as well drop this as well:
```suggestion
```

Reply:
Or not, not sure why I thought that.

Reply:
Well, it lets you write `class extends OVR_multiview2 { }` whereas the DOM types don't. I don't know if that's *correct* but if the types aren't broken I'm going to let an expert fix it instead of me.